packages/bpk-component-pagination/src/BpkPaginationPage.js

```diff
@@ -25,20 +25,23 @@ import STYLES from './bpk-pagination-page.scss';
 
 const getClassName = cssModules(STYLES);
 
+const computeClassName = isSelected => [
```

Comment:
This is a very different approach from what we usually do. Why?

Reply:
No particular reason, I like having a function that returns a ready to use string e.g. I don't like 
https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-accordion/src/BpkAccordion.js#L34
with the join call in the JSX block

Do you want me to change it?

Reply:
I've always preferred it the other way, with the join call in the JSX block. 

I like that you start with an array that is a useful data structure that *could* be used elsewhere at some point, versus the string which is only useful in the JSX block. And then it's the render function's job to convert that to the format it needs.

Reply:
Consistency matters above all else 👍

Reply:
https://media.giphy.com/media/sJPJTSbXSevxS/giphy.gif